### ISSUE TITLE
Fix issue with dead weights

### ISSUE THREAD

**Issue by mariushe** [mariushe/convts#1]
When the derivative of the loss function return a negative number we often end up with dead weights because of the Relu function in the backpropagation. Try to fix this by using leaky Relu
